Add 2018 project pages

danford1 opened the pull request:
## Overview

- adds 6 new project pages to the site, 1 for each 2018 project
- adds 2018 fellows
- edits Featured Projects that appear on the Projects page

### Notes

As the 2018 project blogs are published on azavea.com, we will need to add links on the associated project page on this site.

## Testing Instructions

 * git pull
 * Review Projects page and note updated Featured Projects
 * Review /projects/all and note 2018 Projects section
 * Review each 2018 project page, checking quality/accuracy of text, images, and links

eneedham commented on _data/fellows.yml:
```diff
@@ -1,3 +1,18 @@
+gabrielleBeaudry:
+  name: Gabrielle Beaudry
+  info: 'Master of Science candidate in Geographic Information Science for Development and Environment at Clark University'
```
wrong info

danford1 replied:
Updated.

eneedham commented on _data/fellows.yml:
```diff
@@ -1,3 +1,18 @@
+  image: gabrielleBeaudry.jpg
+xiaoWu:
+  name: Xiao Wu
+  info: 'Master of Science candidate in Geographic Information Science for Development and Environment at Clark University'
```
wrong info

danford1 replied:
Updated.

eneedham commented on _data/fellows.yml:
```diff
@@ -1,3 +1,18 @@
+  image: xiaoWu.jpg
+daleWatt:
+  name: Dale Watt
+  info: 'Master of Science candidate in Geographic Information Science for Development and Environment at Clark University'
```
exact same info as Xiao and Gab - looks like it also matches Sarah Gates. This could be right for Dale, not sure.

danford1 replied:
Updated.

danford1 commented:
- [x] updated all fellow info based on their resumes.
- [x] updated the project description text based on more-recent edits made to the Google doc.
- [x] changed brief_description text for all projects to sentence case

The 2018 project shortlist was already live on the site; this edit indicates which 2018 projects were selected/completed.

eneedham commented on _data/fellows.yml:
```diff
@@ -1,11 +1,11 @@
 gabrielleBeaudry:
   name: Gabrielle Beaudry
-  info: 'Master of Science candidate in Geographic Information Science for Development and Environment at Clark University'
+  info: 'BS Environmental Science, Energy Management track; Minor in Computer Science at UNC Chapel Hill'
```
You might want to include her master's that she is starting:

BS Environmental Science, Energy Management track; Minor in Computer Science, Master's in Environmental Informatics Candidate at UNC Chapel Hill

eneedham commented on _data/fellows.yml:
```diff
@@ -1,11 +1,11 @@
 xiaoWu:
   name: Xiao Wu
-  info: 'Master of Science candidate in Geographic Information Science for Development and Environment at Clark University'
+  info: 'Master of Urban Spatial Analytics and Landscape Architecture and Regional Planning at the University of Pennsylvania School of Design'
```
candidate: 
'Master of Urban Spatial Analytics and Landscape Architecture and Regional Planning Candidate at the University of Pennsylvania School of Design'

designmatty commented:
A lot of the images in this PR are pretty massive. Im going to pull this down and play with resizing things.
